test/server.js

```diff
@@ -0,0 +1,3 @@
+const TrailsApp = require('trails')
```

Comment:
Could you delete this file please ? It's not needed all is already in `index.js`